src/cljs/main/broadfcui/main.cljs

```diff
@@ -73,13 +66,9 @@
                               {:update-registration? true
                                :on-done #(.. js/window -location (reload))})
           :registered
-          (if (and (= page :status) (config/debug?))
-            (status-page/render)
-            [:div {}
-             (let [item (first (filter #(= (% :key) page) routes))]
-               (if item
-                 ((item :render) {:nav-context nav-context})
-                 [:div {} "Page not found."]))]))]))
+          (if component
+            [component (make-props)]
+            [:h2 {} "Page not found."]))]))
```

Comment:
You could do a Levenshtein distance to all known URLs.  Probably way more effort than its worth though.